tests/lib/tools/suite/tests.cleanup/task.yaml

```diff
@@ -0,0 +1,74 @@
+summary: tests for the tests.cleanup tool
+systems: [ubuntu-16.04-64]
+prepare: |
+    apt-get install -y shellcheck
```

Comment:
I added the shellcheck dependency for ubuntu-18.04-64 in pakgdb helper so the it will be preinstalled, so, you could do the same and no need to install/remove shellcheck on each test

Reply:
Thanks, I'll look at how that was done and do the same here.